src/Tools.js

```diff
@@ -2,6 +2,33 @@
  * Class to hold some global helper tools
  */
 class Tools {
+	/**
+	 * Output into the console.
+	 *
+	 * @param  {string} [type='info'] Log type used for console output
+	 *  valid options are:
+	 *  - 'info' for console.info()
+	 *  - 'warn' for console.warn()
+	 *  - 'error' for console.error()
+	 * @param  {string} str String to output
+	 * @param {Mixed} [params] Parameters to output into the console
+	 *  alongside the message. This is especially useful for errors.
+	 */
+	static log( type = 'info', str = '', params ) {
```

Comment:
Yeah good point, I'll change that

Reply:
Yeah and we should probably examine whether the `error` we're outputting on the API error is actually an error (it just notifies us, it's more debug) especially since I noticed Chrome considers it an error in the extension log :(